src/components/House/House.js

```diff
@@ -9,51 +9,36 @@ import { useEffect, useState } from 'react';
 
 function House() {
   const { id } = useParams();
-  const [house, setHouse] = useState({});
-  const [dropdownTabEquipement, setDropdownTabEquipement] = useState({});
-  const [dropdownTabDescription, setDropdownTabDescription] = useState({});
-  const [tags, seTags] = useState([]);
-  const [rating, setRating] = useState([]);
-
-  // useEffect(() => {
-  //   fetch('/data/data.json')
-  //     .then((response) => response.json())
-  //     .then((data) => {
-  //       const house = data.find((house) => house.id === id);
-  //       setHouse(house);
-  //       dropdownTabDetail = {title:'équipement', content:house.equipements}
-  //     });
-  // }, [id]);
+  const [house, setHouse] = useState();
 
   useEffect(() => {
     const getHouse = async () => {
       const response = await fetch('/data/data.json');
       const data = await response.json();
       const house = data.find((house) => house.id === id);
+
       setHouse(house);
-      const equipmentHTML = house.equipments.map((equipment) => <li>{equipment}</li>);
-      setDropdownTabEquipement({ header: 'Équipements', body: equipmentHTML });
-      setDropdownTabDescription({ header: 'Description', body: house.description });
-      let tagsHTML = house.tags.map((tag) => (
-        <button key={tag} className="House__cardPartOne__button">
-          {tag}
-        </button>
-      ));
-      seTags(tagsHTML);
-      let ratingHTML = [];
-      for (let i = 0; i < 5; i++) {
-        if (i < house.rating) {
-          ratingHTML.push(<img key={i} src={starActive} className="House__star" alt="star" />);
-        } else {
-          ratingHTML.push(<img key={i} src={starInactive} className="House__star" alt="star" />);
-        }
-      }
-      console.log(ratingHTML);
-      setRating(ratingHTML);
     };
     getHouse();
   }, [id]);
 
+  if (!house) {
+    return null;
+  }
+
+  let ratingElements = [];
+  for (let i = 0; i < 5; i++) {
+    if (i < house.rating) {
+      ratingElements.push(
+        <img key={i} src={starActive} className="House__star" alt="star" />
+      );
+    } else {
+      ratingElements.push(
+        <img key={i} src={starInactive} className="House__star" alt="star" />
+      );
+    }
+  }
+
```

Comment:
tu peux simplifier ça avec quelque chose du genre :

```suggestion
  let ratingElements = [];
  for (let i = 0; i < 5; i++) {
    const imgSrc = (i < house.rating) ? starActive : starInactive;
    ratingElements.push(
       <img key={i} src={imgSrc} className="House__star" alt="star" />
   );
  }
```